guides/upgrade_to_2_0.md

```diff
@@ -0,0 +1,105 @@
+Upgrading to Bamboo 2.0
+=======================
+
+Bamboo 2.0 ships with some breaking changes (hence the major version bump). But
+don't worry, if you love Bamboo just as it is, there's an easy upgrade path.
+
+There are two breaking changes:
+
+- `Bamboo.Phoenix` extracted to [bamboo_phoenix]
+- `Bamboo.Mailer.deliver_now/2` and `deliver_later/2` don't raise on errors
+
+Let's cover each in turn.
+
+## Breaking change: `Bamboo.Phoenix` extracted to `bamboo_phoenix`
+
+`Bamboo.Phoenix` has been extracted to the [bamboo_phoenix] library. If you use
+`Bamboo.Phoenix` to render your email templates, add `bamboo_phoenix` to your
+dependencies:
+
+```elixir
+defp deps do
+  [
+    ...
+    {:bamboo, "~> 2.0"},
+    {:bamboo_phoenix, "~> 1.0"}
+    ...
+  ]
+end
+```
+
+## Breaking change: `deliver_now/2`/`deliver_later/2` return `:ok` & `:error` tuples
+
+`Bamboo.Mailer`'s `deliver_now/2` and `deliver_later/2` no longer raise errors.
+Instead, they now return an `{:ok, email}` and `{:error, error}`, where the
+`error` is an exception struct or an error message. If you pass `response:
+true` as an argument, the return value will be `{:ok, email, response}`.
+
+If you prefer seeing code, this is the change in `@spec` signature for
+`deliver_now/2`:
+
+```diff
+-  @spec deliver_now(Bamboo.Email.t(), Enum.t()) :: Bamboo.Email.t() | {Bamboo.Email.t(), any}
++  @spec deliver_now(Bamboo.Email.t(), Enum.t()) ::
++          {:ok, Bamboo.Email.t()}
++          | {:ok, Bamboo.Email.t(), any}
++          | {:error, Exception.t() | String.t()}
+```
+
+Note that `deliver_later/2` will only return errors that happen _prior_ to
+scheduling the delivery of the email. What happens once the delivery is
+scheduled depends on what [delivery strategy] you are using.
+
+Those who want to handle errors on their own can now pattern match on the `:ok`
+and `:error` tuple responses. If you don't want to handle the errors and like
+how Bamboo behaves prior to 2.0, there's a simple upgrade path. 👇
+
+### Simple upgrade path
+
+`Bamboo.Mailer` comes with `deliver_now/2!` and `deliver_later/2!`. Those two
```

Comment:
Should this be 
```suggestion
`Bamboo.Mailer` comes with `deliver_now!/2` and `deliver_later!/2`. Those two
```
here and in other locations

Reply:
Thank you. Great catch! I updated that in e31f997a7cc41379301141aa19f3c55a1f8359b1